pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -647,11 +647,6 @@ func (r *Builder) setMachine(object *cnv.VirtualMachineSpec) {
 func (r *Builder) mapMemory(vm *model.VM, object *cnv.VirtualMachineSpec) {
 	memoryBytes := int64(vm.MemoryMB) * 1024 * 1024
 	reservation := resource.NewQuantity(memoryBytes, resource.BinarySI)
-	object.Template.Spec.Domain.Resources = cnv.ResourceRequirements{
-		Requests: map[core.ResourceName]resource.Quantity{
-			core.ResourceMemory: *reservation,
-		},
-	}
 	object.Template.Spec.Domain.Memory = &cnv.Memory{Guest: reservation}
```

Comment:
`reservation` is this the real memory size of the guest?

Or something like ballooned memory.

Reply:
the real memory size of the guest

Reply:
+1